ui-v2/tests/acceptance/components/copy-button.feature

```diff
@@ -0,0 +1,14 @@
+@setupApplicationTest
+Feature: components / copy-button
+  Background:
+    Given 1 datacenter model with the value "dc-1"
+  Scenario: Clicking the copy button
+    When I visit the node page for yaml
+    ---
+      dc: dc-1
+      node: node-0
+      ---
+    Then the url should be /dc-1/nodes/node-0
+    When I click ".healthcheck-output:nth-child(1) button.copy-btn"
+    Then I see the text "Sorry, something went wrong!" in ".healthcheck-output:nth-child(1) p.feedback-dialog-out"
+    Then I see the text "Copied output!" in ".healthcheck-output:nth-child(1) p.feedback-dialog-out"
```

Comment:
This is what I originally expected the feedback dialog of the copy button to say, as it's what that value is when running the demo frontend with `make start-api` and `make start`.